src/pages/project/userSettings.tsx

```diff
@@ -0,0 +1,83 @@
+import Header from "@/src/components/layouts/header";
+import { Alert } from "@/src/components/ui/alert";
+import { useSession, signOut } from "next-auth/react";
+import { LogOut } from "lucide-react";
+import { ChevronRightIcon } from "@heroicons/react/20/solid";
+import {
+  Avatar,
+  AvatarFallback,
+  AvatarImage,
+} from "@/src/components/ui/avatar";
+
+const instructionItems = [
+  {
+    name: "LogOut",
+    description: "Click button to safely end your current session now",
+    onClick: () =>
+      signOut({
+        callbackUrl: "/auth/sign-in",
+      }),
+    icon: LogOut,
+  },
+];
+
+export default function UserSettingPage() {
+
+  const session = useSession()
+
+  return (
+    <div className="md:container">
+      <Header title="User Settings" />
+      <Alert>
+        <div className=" flex justify-between items-center ">
+          <div className="text-2xl font-bold">{session.data?.user?.name}</div>
+          <Avatar className="h-20 w-20">
+            <AvatarImage
+              src={session.data?.user?.image ?? undefined}
+            />
+            <AvatarFallback className="h-20 w-20 text-2xl">
+              {session.data?.user?.name
+                ? session.data.user.name
+                  .split(" ")
+                  .map((word) => word[0])
+                  .slice(0, 2)
+                  .concat("")
+                : null}
+            </AvatarFallback>
+          </Avatar>
+        </div>
+        <ul
+          role="list"
+          className="mt-6 divide-y divide-gray-200 border-b border-t border-gray-200"
+        >
+          {instructionItems.map((item, itemIdx) => (
+            <li key={itemIdx}>
```

Comment:
The UI here looks a bit large to me. Can you make sizings similar to the sizes on the left side bar?

Reply:
I'm sorry, I don't quite understand. Could you explain a bit more?
![image](https://github.com/langfuse/langfuse/assets/94478026/e6e3892b-bec0-46fb-ac71-f5618555bf72)
i decrease little bit size of user name